tests/gocase/unit/scan/scan_test.go

```diff
@@ -52,13 +52,26 @@ func TestScanEmptyKey(t *testing.T) {
 	require.Equal(t, []string{"", "fab", "fiz", "foobar"}, keys)
 }
 
-func TestScan(t *testing.T) {
+func TestScanWithNumberCursor(t *testing.T) {
 	srv := util.StartServer(t, map[string]string{})
 	defer srv.Close()
+	ctx := context.Background()
+	rdb := srv.NewClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
+	ScanTest(t, rdb, ctx)
+}
 
+func TestScanWithStringCursor(t *testing.T) {
+	srv := util.StartServer(t, map[string]string{})
+	defer srv.Close()
 	ctx := context.Background()
 	rdb := srv.NewClient()
 	defer func() { require.NoError(t, rdb.Close()) }()
+	require.NoError(t, rdb.ConfigSet(ctx, "redis-cursor-compatible", "no").Err())
+	ScanTest(t, rdb, ctx)
+}
```

Comment:
The default value of `redis-cursor-compatible` is `no`. I think the `TestScanWithNumberCursor` does not work as expected.

Reply:
Oh, I changed the default value of `redis-cursor-compatible` but forgot to update the test.